controllers/nginx/pkg/cmd/controller/metrics.go

```diff
@@ -40,7 +42,10 @@ func (em exeMatcher) MatchAndName(nacl common.NameAndCmdline) (bool, string) {
 }
 
 func (n *NGINXController) setupMonitor(args []string) {
-	pc, err := newProcessCollector(true, exeMatcher{"nginx", args})
+	var enableVts = true
```

Comment:
Ok, so what method is the recommend to call the setupMonitor? Is the reload method?

```
func (n NGINXController) Reload(data []byte) ([]byte, bool, error) {
```